data/main.ts

```diff
@@ -163,10 +163,16 @@ const setupInfrastructure = async (): Promise<void> => {
         }
       );
 
+      new S3Object(this, "main_agent_js", {
+        bucket: mainWebsite.bucket,
+        key: "scripts/agent.js",
+        source: "./data/scripts/agent.js",
```

Comment:
any public downloadable scripts will live in `https://samepage.network/scripts/*`